.github/workflows/workflow.yml

```diff
@@ -233,6 +233,94 @@ jobs:
           # We disable the Dune cache when running the tests
           DUNE_CACHE: disabled
 
+  wasm:
+    name: Wasm_of_ocaml
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install node
+        uses: actions/setup-node@v4
+        with:
+          node-version: v22.0.0-v8-canary2024030314ed92e804
+
+      - name: Restore cached binaryen
```

Comment:
Are these "name:" fields like titles?

Reply:
I'm not sure what you mean.

When a PR is modified, some jobs are executed to check that everything is fine. A job contains a sequence of tasks called steps. This is [the name of a step](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsname).

It appears here: https://github.com/ocaml/dune/actions/runs/9820743026/job/27115667595
![image](https://github.com/user-attachments/assets/8f430e88-574d-43bf-854e-2f7b2eec6b06)

Reply:
Thank you for the explanation and visual. That helps!

Reply:
Since they appear in that list, they're section headers, so they definitely should use title case. I'm not able to make suggestions on lines before 197, but they should all be title case. The only word that I can think of not capitalized in a title is opam and `odoc`, the latter always appearing in code block. Otherwise, if the word in the title is a command, you can keep it lower case and wrap it in monospace code block.